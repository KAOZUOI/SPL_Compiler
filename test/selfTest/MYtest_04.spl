struct Book {
  int id;
  char title;
};

struct Book {
  float price;
  int pages;
};

int printBook(struct Book book) {
  return book.id;
}

int main() {
  struct Book myBook;
  myBook.id = 123;
  myBook.title = 'C';
  printBook(123);
}
