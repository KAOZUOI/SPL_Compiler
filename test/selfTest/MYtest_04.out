Error type 15 at Line 6: redefine the same structure type "Book".
Error type 9 at Line 19: the function "printBook" is not applicable for arguments.